source/admin_manual/list_of_events.rst

```diff
@@ -982,6 +982,38 @@ Mail user
 | user                | Username of the user                                 |
 +---------------------+------------------------------------------------------+
 
+mail_user_session_finished
+--------------------------
+.. versionadded:: v2.3.18
+
++---------------------+------------------------------------------------------+
+| Field               | Description                                          |
++=====================+======================================================+
+| Inherits from :ref:`event_mail_user`                                       |
++---------------------+------------------------------------------------------+
+| stime               | System CPU time used                                 |
++---------------------+------------------------------------------------------+
+| minor_faults        | Page reclaims (soft page faults)                     |
++---------------------+------------------------------------------------------+
+| major_faults        | Page faults (hard page faults)                       |
++---------------------+------------------------------------------------------+
+| vol_cs              | Voluntary context switches                           |
++---------------------+------------------------------------------------------+
+| invol_cs            | Involuntary context switches                         |
++---------------------+------------------------------------------------------+
+| rss                 | Integral unshared data size                          |
++---------------------+------------------------------------------------------+
+| vsz                 | Virtual memory size                                  |
++---------------------+------------------------------------------------------+
+| rchar               | Characters read                                      |
++---------------------+------------------------------------------------------+
+| wchar               | Characters written                                   |
```

Comment:
The rchar, wchar, syscr, syscw explanations could just be copy & pasted from https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/filesystems/proc.rst?id=HEAD#n1730